emission/net/api/stats.py

```diff
@@ -46,22 +46,33 @@ def storeClientEntry(user, key, ts, reading, metadata):
   currEntry = createEntry(user, key, ts, reading)
   # Add the os and app versions from the metadata dict
   currEntry.update(metadata)
-  get_client_stats_db().insert(currEntry)
+  response = get_client_stats_db().insert(currEntry)
```

Comment:
Yes. You make it fail, maybe by giving a bad/invalid input, or by
reconfiguring the URL within the test, and then ensuring that nothing is
raised. Then you can restore the original URL.